public/scripts/extensions/tts/index.js

```diff
@@ -105,6 +106,7 @@ const ttsProviders = {
     System: SystemTtsProvider,
     VITS: VITSTtsProvider,
     XTTSv2: XTTSTtsProvider,
+    'TTS WebUI (Unofficial)': TtsWebuiProvider,
```

Comment:
Why Unofficial? Are you referring to someone's fork?

Reply:
I thought unofficial indicates providers written by non-ST members. In this case, it is the official one.
```suggestion
    'TTS WebUI': TtsWebuiProvider,
```

Reply:
Unofficial = the upstream doesn't have a usable API endpoint, but there exists a fork which adds one, and the provider only works for that fork.